user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -0,0 +1,281 @@
+
+> [!TIP]
+> See also:
+>
+> - [Kata #19: Transform low-code apps into visual delights](https://community.dataminer.services/courses/kata-19/) on DataMiner Dojo ![Video](~/user-guide/images/video_Duo.png)
+> - Tutorial: [Creating a visually appealing and user-friendly low-code app](xref:Tutorial_App_Design)
+
+## Colors/themes
+
+- When you create a new app or dashboard, it essential to start from a good **theme**. You can for example [duplicate a dashboard](xref:Duplicating_a_dashboard) or [duplicate an app](xref:Creating_custom_apps#duplicating-an-existing-low-code-app) with a similar look and feel to what you have in mind, and then fine-tune [the dashboard layout](xref:Configuring_the_dashboard_layout) or the [app layout](xref:LowCodeApps_Layout).
+
+- Each theme **must** have a component style with a transparent background. Use this component style for titles, images, and buttons.
+
+  ![Transparent theme](~/user-guide/images/LCA_Style_Guide_Theme_Transparent.png)
+
+- When you define a background color for pages/panels (as part of the theme), never use full white (#FFFFFF) or full black (#000000). Here are some good default colors:
+
+   |☀️  Light background |🌙 Dark background|
+   |---------------------|----------------|
+   |#F4F4F5              |#1E2530         |
+
+- Make sure the background color of your component style is different from the page/panel background. This way, components can easily be distinguished from the background. This can be used for query filters, tables, timelines, etc. (anything but titles, images, and buttons). Generally, it is best to choose a slightly lighter color than that of the page/panel background. Here are some good default colors:
+
+   |☀️  Light background |🌙 Dark background|
+   |------------------|----------------|
+   |#FDFDFD           |#272E38        |
+   |![Component light style](~/user-guide/images/LCA_Style_Guide_Component_Style_Light.png)|![Component dark style](~/user-guide/images/LCA_Style_Guide_Component_Style_Dark.png)|
+
+- When you pick colors for pages, panels, buttons, grids, and tables, there **must** be enough contrast between text and icons and their background. Otherwise, users will have trouble seeing the text and icons.
+
+   |👍 Good |👎 Bad |
+   |------|-----|
+   |![Good contrast](~/user-guide/images/LCA_Style_Guide_Contrast_Good.png)|![Bad contrast](~/user-guide/images/LCA_Style_Guide_Contrast_Bad.png)|
+
+- When you pick the main color for a low-code app, also make sure there is enough contrast. When you select an app color, the app itself will determine what the color of the text is, based on the selected background color. Be sure to pick a color that results in enough contrast between the text and background color, especially in the pages menu on the left, as this will be a slightly darker color than the selected app color.
+
```

Comment:
There were some good suggestions in the following post. Maybe we want to evaluate which one(s) we share?

https://teams.microsoft.com/l/message/19:431bace991e24c0db9c3c99805cd2baf@thread.skype/1677579785217?tenantId=5f175691-8d1c-4932-b7c8-ce990839ac40&groupId=3ccc5562-e346-4a6f-a03a-3db73cf0475e&parentMessageId=1677579785217&teamName=Skyline%20-%20Global&channelName=The%20Art%20of%20AI%20%F0%9F%A4%96&createdTime=1677579785217

Reply:
for me these autogenerated palettes never gave the desired result, so I personally do not recommend. but if other people have a good experience with it please add it